### PR TITLE
Continue evolution of Survivor Armor take 3.

### PR DESCRIPTION
**Pull request by keampe**
#### Summary
Balance "Update and refine Kevlar Jumpsuits for the Tailoring overhaul. Third times the charm, I hope"

#### Purpose of change
The tailoring overhaul made fabrics more consistent but the survivor armor wasn't updated with the new materials.

#### Describe the solution
Changed the suits and recipes for the jumpsuits to use the more accurate material amounts, then took a further look.

Some other changes

    1. The light Kevlar jumpsuit, Kevlar jumpsuit and heavy Kevlar jumpsuit reduced the Kevlar layer by 0.3mm and gave them a 0.3mm layer of Synthetic Fabric keeping the same overall thickness. Why? Kevlar degrades in sunlight. It's the reason why all Kevlar products meant for outdoors applications have a covering or are embedded into something (like racing tires). I assume that the fur, faux fur, and neoprene have an outer layer of their respective materials. Of note: Nomex also degrades in sunlight - but synthetic fabric is nylon - and that's a terrible coating for flame resistant gear. I left it alone - Don't know what to do about it. Also note: Kevlar does come in coated UV resistant varieties, but the survivor doesn't get to pick and choose often.
    2. Removed the raincoat or equivalent from the crafting recipes. Doesn't need it.
    3. Removed the Duct Tape from the crafting recipes. Duct tape, while I'm a huge fan of it in general, is not part of permanent clothing.
    4. Added in fasteners. Lots of them. These are to keep the fit close to the body in some areas while allowing freedom of movement. For example keeping the forearms and upper torso snug while leaving the shoulders, upper arms and elbows free gives good arm movement while keeping the cuff closed. Also, this is a [sturdy] garment and closures are a weak point - they're re-enforced.
    5. Removed the [waterproof] flags. The Nylon layers is water resistant, so I left the [rainproof] flag. Just don't go swimming in it. :)
    6. Adjusted the amounts of materials used for Kevlar to be more appropriate - and changed them from layered Kevlar panels to Kevlar sheets where they were still layered panels.
    7. Kevlar thread - used in these recipes - is often of the coated variety. I left it for this reason.

#### Describe alternatives you've considered
Adjusting the thickness. These suits are very thick for the normal layer. The "Light" Kevlar jumpsuit has a minimum thickness of 1.5mm over 10% of covered areas and 2.5mm thickness over 90% of covered areas. The regular Kevlar Jumpsuit is 2.2mm thick minimum - but this is only over 1% of it's coverage. 18% is 3.2mm and 81% is 4.2mm. This is way over the 2mm guidelines listed in the armor balance doc. Still, they passed earlier muster and wetsuits are often as thick and one could wear a jacket over them.

#### Testing
Modded my game loaded up and all recipes showed up. No errors in loading.

#### Additional context
Third try, hope this works.

### CODE REVIEW

**Review by bombasticSlacks — APPROVED** [review]
gotta adjust the materials in the top level of the item to match the armor (so add nylon) or delete those lines entirely.

Otherwise this looks good to me

**Comment by keampe** [comment]
Done, and thanks for all the help!

   - Shane

**Comment by keampe** [comment]
Not sure why the GCC9,Curses, LTO test is failing.

   - Shane

**Comment by anothersimulacrum** [comment]
Not your fault https://github.com/CleverRaven/Cataclysm-DDA/runs/7433550642?check_suite_focus=true#step:16:498

**Comment by keampe** [comment]
Ah... Thank you.

   - Shane

**Comment by keampe** [comment]
Is there more you want me to do on this, or just leave it another week - happy either way. :)

   - Shane

**Comment by bombasticSlacks** [comment]
I was gonna look at this later today. Need to check that the materials weight lines up with the weights of the suits and that the materials are all represented in the crafting recipes.

**Comment by keampe** [comment]
Cool! Thank you! I am willing to make any changes you want, including the thickness of the Kevlar if you want to put these more in line with the Armor_Balance doc - or not if you don't feel it necessary.

   - Shane

File: data/json/recipes/armor/bespoke_armor/survivor.json [hunk 1]
```diff
@@ -9,48 +9,35 @@
...
       { "proficiency": "prof_closures_waterproofing" },
       { "proficiency": "prof_polymerworking" },
       { "proficiency": "prof_elastics" }
     ],
-    "components": [
-      [ [ "coat_rain", 1 ], [ "jacket_windbreaker", 1 ], [ "jacket_evac", 1 ], [ "coat_gut", 1 ] ],
-      [ [ "duct_tape", 300 ] ],
-      [ [ "sheet_kevlar_layered", 24 ] ]
-    ]
+    "components": [ [ [ "sheet_kevlar", 130 ] ] ]
```
Comment (bombasticSlacks) [inline data/json/recipes/armor/bespoke_armor/survivor.json:20]: These seem really low as values. A layered Kevlar panel is 16 sheets, so these counts should be close to 16 * 24 (in this case). So at least double the cost of Kevlar, realistically close to 3 times.

File: data/json/recipes/armor/bespoke_armor/survivor.json [hunk 2]
```diff
@@ -114,43 +89,31 @@
     "difficulty": 5,
     "time": "6 h",
     "autolearn": true,
-    "using": [ [ "sewing_kevlar", 100 ], [ "fabric_lycra", 20 ] ],
+    "using": [ [ "sewing_kevlar", 100 ], [ "fabric_lycra", 20 ], [ "fabric_nylon", 30 ], [ "fastener_large", 1 ], [ "clasps", 8 ] ],
```
Comment (bombasticSlacks) [inline data/json/recipes/armor/bespoke_armor/survivor.json:92]: why more nylon than lycra? if the nylon is an other layer and the lycra is an inner layer shouldn't they be approximately equivalent materials wise.

**Review by bombasticSlacks — COMMENTED** [review]
overall looks good just a few thoughts on used materials

File: data/json/recipes/armor/bespoke_armor/survivor.json (hunk 1 above)
Comment (keampe) [reply data/json/recipes/armor/bespoke_armor/survivor.json:20]: Part of why I did the PR is that after the tailoring audit the materials all weighed more. A Layered Kevlar Panel is, after the tailoring audit, 429g so 24 of them would be 10.296kg. This is more than the suit weighs by a large factor. the entire suit including all materials weighs only 8.4kg. the Faux Fur sheets themselves weigh in at 372g apiece for 7.44kg. The total is more than twice the weight of the final product and while there should be some wastage 50% is too much. Either I have to reduce the materials or increase the weight of the final product.

I can do either, BTW, but I chose to reduce the mats.

   - Shane

File: data/json/recipes/armor/bespoke_armor/survivor.json (hunk 2 above)
Comment (keampe) [reply data/json/recipes/armor/bespoke_armor/survivor.json:92]: The Nylon layer is 0.3mm and the Lycra is 0.2mm so I thought 1.5 times seemed reasonable. :) I can change them if you like.

   - Shane

**Pull request merged**